### PR TITLE
Add SDQL, +PROCCALL, admin updates

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Adds SDQL proc calls, which lets you (conditionally) call several procs on specified objects. By Mloc: https://github.com/Baystation12/Baystation12/pull/3871.
2) Adds a new permission, +PROCCALL. Advanced/Atom Proc Call and SDQL are gated behind this.
3) Adds proper rights checks/logging to various admin procs.
4) Gets rid of unused example config files.
5) Mods once again have access to the traitor panel.
6) Fixes chems irradiating you taking armor into account.
7) All admins can now rename players.

### CODE REVIEW

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -754,6 +676,9 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	set name = "Gibself"
 	set category = "Event"
 
+	if(!check_rights(R_ADMIN|R_EVENT))
```
Comment (tigercat2000) [inline code/modules/admin/verbs/randomverbs.dm]: Does this mean you need to have R_ADMIN and R_EVENT, or R_ADMIN or R_EVENT, for clarification?

File: code/modules/admin/verbs/randomverbs.dm [hunk 2]
```diff
@@ -764,112 +689,26 @@ Traitors and the like can also be revived with the previous role mostly intact.
...
 /client/proc/toggle_view_range()
 	set category = "Special Verbs"
 	set name = "Change View Range"
 	set desc = "switches between 1x and custom views"
+	
+	if(!check_rights(R_ADMIN))
+		return
 
 	if(view == world.view)
 		view = input("Select view range:", "View Range", 9) in list(1,2,3,4,5,6,7,8,9,10,11,12,13,14,128)
```
Comment (tigercat2000) [inline code/modules/admin/verbs/randomverbs.dm]: Minor thing, could you change the default here to 7?

Comment (PsiOmegaDelta) [inline code/modules/admin/verbs/randomverbs.dm]: Doesn't world.view have the default view range?

File: code/modules/admin/verbs/randomverbs.dm (hunk 1 above)
Comment (Markolie) [inline code/modules/admin/verbs/randomverbs.dm]: It's OR.

File: code/modules/admin/verbs/randomverbs.dm (hunk 2 above)
Comment (tigercat2000) [inline code/modules/admin/verbs/randomverbs.dm]: That would also probably work fine.

**Comment by JJRcop** [comment]
ACKhem... *ak* **Achaaha**chem *AHEM*

for the record I made the multiquery part, but wasn't attributed here, and VG attributed it to you since you didn't attribute me :smile:

also this looks like it was ported from tg not bay

:smile: :smile: :joy:

**Pull request merged**